bin/cancel-pending-tx.js

```diff
@@ -0,0 +1,172 @@
+// dotenv must be imported before importing anything else
+import 'dotenv/config'
+
+import assert from 'node:assert'
+import { ethers } from 'ethers'
+import { CoinType, newDelegatedEthAddress } from '@glif/filecoin-address'
+
+import { createMeridianContract } from '../lib/ie-contract.js'
+
+const {
+  WALLET_SEED
+} = process.env
+
+const [, , tx] = process.argv
+
+assert(WALLET_SEED, 'WALLET_SEED required')
+assert(tx, 'Transaction hash must be provided as the first argument')
+
+const { provider } = await createMeridianContract()
+
+const signer = ethers.Wallet.fromPhrase(WALLET_SEED, provider)
+const walletDelegatedAddress = newDelegatedEthAddress(/** @type {any} */(signer.address), CoinType.MAIN).toString()
+console.log(
+  'Wallet address:',
+  signer.address,
+  walletDelegatedAddress
+)
+
+console.log('Going to revoke %s', tx)
+
+const messages = await getMessagesInMempool(walletDelegatedAddress)
```

Comment:
Double checking: The mempool is the only place we can look for pending messages?